### ISSUE TITLE
backend exceptions should not be sent to user

### ISSUE THREAD

**Issue by volodymyrss** [oda-hub/dispatcher-app#617]
OSError: [Errno 28] No space left on device: \\\'/scratch/oda-dda-interface-59df8cc467-vz8hf/ISGRILCSum/2023-10/25/09-11-03-219-68ba5dbb

**Comment by burnout87** [comment]
example:

![image](https://github.com/oda-hub/dispatcher-app/assets/3757833/977367d0-5a2f-432b-9836-158139624967)

**Comment by volodymyrss** [comment]
yeah, this should be sent to the user. Only to sentry.

**Comment by volodymyrss** [comment]
Yes. But we should at least wrap something reasonable before showing.

**Comment by burnout87** [comment]
With #626 , now in the  frontend we receive the following:

![image](https://github.com/oda-hub/dispatcher-app/assets/3757833/ec123b4b-9167-4be1-b7dc-10c3d3cf6ba1)

and on sentry, a more detailed message (along with the other exceptions):

![image](https://github.com/oda-hub/dispatcher-app/assets/3757833/51702757-0bdf-4c40-8712-8bbf03bcb64c)

**Comment by volodymyrss** [comment]
> ![image](https://private-user-images.githubusercontent.com/3757833/287711281-3c7a5136-a1a8-487a-8ec9-f244370329c2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDE4MDQ0NDUsIm5iZiI6MTcwMTgwNDE0NSwicGF0aCI6Ii8zNzU3ODMzLzI4NzcxMTI4MS0zYzdhNTEzNi1hMWE4LTQ4N2EtOGVjOS1mMjQ0MzcwMzI5YzIucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQUlXTkpZQVg0Q1NWRUg1M0ElMkYyMDIzMTIwNSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyMzEyMDVUMTkyMjI1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NGEzZDg3ZjFjNWNlZDQ1NTIzZTM1ZDkzMGQ3NDc5NzJmZGVlMjc0NjE4ZGY0ZTM4MDM3M2YxNGJkNDg4YzM5ZiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.cCt-pLuX-Dtk_ZmZd9AILC45MSEQe_0NDWxPK90gqCs)

Thanks! Oh, another detail, looking on the screenshot, we could add  "If you wish to provide further feedback, please use "Write Feedback" button below. We will make sure to respond to any feedback provided!"

**Comment by burnout87** [comment]
better:

![image](https://github.com/oda-hub/dispatcher-app/assets/3757833/77dc6ebd-7a85-41f5-b8b8-14490ab7c465)
